plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1753,4 +1756,345 @@ private Set<String> getAllFilesFromCdfDirectory(String tableName)
                 .filter(path -> path.contains("/" + CHANGE_DATA_FOLDER_NAME))
                 .collect(toImmutableSet());
     }
+
+    @Test
+    public void testPartitionFilterQueryNotDemanded()
+    {
+        Map<String, String> catalogProperties = getSession().getCatalogProperties(getSession().getCatalog().orElseThrow());
+        assertFalse(Boolean.parseBoolean(catalogProperties.getOrDefault("query_partition_filter_required", "false")));
```

Comment:
Why do we use `getOrDefault` instead of `get`?

Reply:
in case the null is returned when there is no property

Reply:
I'm not sure about your intention, but I don't think `getOrDefault` is suitable in this test. Please consider using the following or something. 
```java
assertThat(catalogProperties).doesNotContainKey("query_partition_filter_required");
```